Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Payment Flow/PaymentConfirmationViewController.m

```diff
@@ -17,6 +19,15 @@ @implementation PaymentConfirmationViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     [self hideNavigationThings:YES];
+    self.confirmationCheckmarkImageView.hidden = YES;
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+    [UIView animateWithDuration:2 animations:^{
```

Comment:
2 seconds might be too long, I would try to stray under a second in general so the app feels fast